File: C/1010.c
#include <stdio.h>

double value_to_pay(double preco1, int quantidade1, double preco2, int quantidade2);

int main(void) {
    int codigo1, codigo2, quantidade1, quantidade2;
    double preco1, preco2, total;
    scanf("%d", &codigo1);
    scanf("%d", &quantidade1);
    scanf("%lf", &preco1);
    scanf("%d", &codigo2);
    scanf("%d", &quantidade2);
    scanf("%lf", &preco2);
    total = value_to_pay(preco1, quantidade1, preco2, quantidade2);
    printf("VALOR A PAGAR: R$ %.2f\n", total);
    return 0;
}

double value_to_pay(double preco1, int quantidade1, double preco2, int quantidade2) {
    return (preco1 * quantidade1) + (preco2 * quantidade2);
}

File: Python/1010.py
codigo1, quantidade1, preco1 = input().split()
codigo1 = int(codigo1)
quantidade1 = int(quantidade1)
preco1 = float(preco1)
codigo2, quantidade2, preco2 = input().split()
codigo2 = int(codigo2)
quantidade2 = int(quantidade2)
preco2 = float(preco2)
total = float((quantidade1 * preco1) + (quantidade2 * preco2))
print(f"VALOR A PAGAR: R$ {total:.2f}")